app/views/providers/address_selections/show.html.erb

```diff
@@ -7,10 +7,10 @@
     ) do |form| %>
 
     <% if @address_collection.count > 1 %>
```

Comment:
Improvement: may be possible to use `.many?` instead of `.count > 1`

Reply:
have made suggested change